extensions/amp-consent/0.1/consent-info.js

```diff
@@ -122,6 +120,9 @@ export function composeStoreValue(consentInfo, opt_forceNew) {
     obj[STORAGE_KEY.STATE] = 1;
   } else if (consentState == CONSENT_ITEM_STATE.REJECTED) {
     obj[STORAGE_KEY.STATE] = 0;
+  } else {
+    // Only store consentString and dirtyBit with reject/accept action
```

Comment:
I don't see where the dirty bit is defined, but I think it may be good (if not already), comment / document that the dirty bit is only set on accept/reject. And not on interaction. Personally, I think of "dirty" input fields, where once you click the field (interaction), but don't actually do anything it, it becomes dirty.

Thus I think leaving a comment on that would help clarify if not there already 😄

Reply:
dirtyBit is not supported yet. I shouldn't have added in last PR😢 Could I add the comment when the support to dirtyBit is implemented, I think it would be much easier.

Reply:
That's cool with me 😄